test/IIS.Tests/MofFileTests.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
```

Comment:
this will only ensure the mof file is well formatted but does not guarantee the content is correctly. Ideally, we should have some e2e test using logon/ETW API to retrieve the event.

Reply:
Follow up PR. The events to day are not great, so I'll add some more and then add E2E tests for them.